src/stories/Library/recommender/recommender.scss

```diff
@@ -0,0 +1,83 @@
+.recommender {
+  background-color: $c-global-tertiary-2;
+
+  @include breakpoint-m {
+    padding: $s-6xl;
+  }
+
+  h1 {
+    color: $c-text-primary-white;
+    text-align: center;
+    padding: $s-lg;
+
+    @include breakpoint-m {
+      padding-bottom: $s-6xl;
+    }
+  }
+}
+
+.recommender-grid {
+  display: grid;
+  grid-template-columns: 50% 50%;
+  grid-template-rows: 50% 50%;
+
+  @include breakpoint-m {
+    grid-template-columns: 33.33% 33.33% 33.33%;
+  }
+}
+
+.recommender__grid__material {
+  border: 1px solid $c-text-secondary-gray;
+  padding: 30px;
+  display: flex;
+  width: 100%;
+  position: relative;
+  height: 350px;
```

Comment:
Consider what will happen if the material has a very long title or list of authors. 

This specifies a fixed height and there does not seem to be anything which tries to tame very long titles or lists of creators.

Reply:
Ill fix